src/context/DictionaryContext.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import dictionary from './../data/dictionary.json';
+
+const DictionaryContext = React.createContext();
+
+export const DictionaryProvider = ({ children }) => {
+  const dictionaryData = {
+    easy: [],
+    medium: [],
+    hard: [],
+  };
+
+  dictionary.forEach((word) => {
+    if (word.length <= 4) {
+      dictionaryData.easy.push(word);
+    } else if (word.length >= 5 && word.length <= 8) {
+      dictionaryData.medium.push(word);
+    } else {
+      dictionaryData.hard.push(word);
+    }
+  });
```

Comment:
Also you can use switch when there are more than 2 ifs. switch is more performant in that way.

Reply:
sure @naquiuddin, will update it